scripts/load_submissions_graph.py

```diff
@@ -0,0 +1,112 @@
+import argparse
```

Comment:
Please, add the requirements and README files

Reply:
Also, don't forget to add a new GitHub action to check styles for Python scripts, an example can be found [here](https://github.com/JetBrains-Research/Lupa/blob/main/.github/workflows/python-build.yml)